docs/_source/getting_started/quickstart_installation.ipynb

```diff
@@ -28,6 +28,7 @@
     "\n",
     "[![deploy on spaces](https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg)](https://huggingface.co/new-space?template=argilla/argilla-template-space)\n",
     "\n",
+    "Note that, Hugging Face spaces don't have data persistent and reset after 48 hour of inactivity. You can enable data persistence via the Hugging Face Spaces settings.\n",
```

Comment:
```suggestion
    "HuggingFace Spaces now have persistent storage and this is supported from Argilla 1.11.0 onwards, but you will need to manually activate it via the HuggingFace Spaces settings. Otherwise, unless you're on a paid space upgrade, after 48 hours of inactivity the space will be shut off and you will lose all the data. To avoid losing data, we highly recommend using the persistent storage layer offered by HuggingFace.\n",
```